files/ovirt-dr

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/python
-from ConfigParser import SafeConfigParser
+try:
```

Comment:
Wouldn't it be better to ask if we are Python 2 or 3 (if six.PY3) and import the relevant version?
The same for other 4 modules.

Reply:
Both methods are fine.
Will stick with this one.